pkg/server/server.go

```diff
@@ -218,6 +220,34 @@ func (s *Server) Run(ctx context.Context) error {
 	return nil
 }
 
+func generateDummyKubeconfig() *rest.Config {
+	return &rest.Config{
```

Comment:
Can this just be `&rest.Config{}` is there any benefit to all the fields? Those seem like all empty values.

Reply:
I'm fine with the method being there it indicates the intent, just seems like it can be a single line

Reply:
Fixed.